ArgosIOS/Argos/Singletons/SocketClient.swift

```diff
@@ -50,6 +51,21 @@ class SocketClient: ObservableObject {
                 print("error", error)
             }
         })
+        
+        self.socket.on("pointMessage", callback: {
```

Comment:
so its still going to be on the message receiver. We actually shouldnt need to edit the socket client. We should be able to sink into the published values in order to actually receive the points themselves.